themes/bootstrap3/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -49,6 +49,11 @@
         <?php endif; ?>
       <?php else: ?>
         <?php /* Begin Unavailable Items (Recalls) */ ?>
+          <?php if ($holding['in_transit'] ?? null): ?>
```

Comment:
I think this code needs to be rewritten a bit -- see lines 20-22 above. This is where the status message and associated class/metadata is set up now, following the refactoring to the availability object. I think instead of having this if/else here, it would make more sense to manipulate those messages earlier -- either by putting conditions in lines 20-22, or by more deeply incorporating this in-transit functionality into the availability object.